frontends/web/src/routes/buy/pocket.tsx

```diff
@@ -91,22 +92,26 @@ export const Pocket = ({ code }: TProps) => {
     // handle address request from moonpay
     try {
       const message = parseMessage(e.data);
-      if (message.type === V0MessageType.RequestAddress && message.withMessageSignature) {
+      if (message.type === V0MessageType.RequestAddress && message.withMessageSignature && !signing) {
+        signing = true;
         const addressType = message.withScriptType ? String(message.withScriptType) : '';
         signAddress(
           addressType,
           String(message.withMessageSignature),
           code)
           .then(response => {
-            if (response.status === 'abort') {
-              // TODO notify to the widget with a message?
-              alertUser('Message signing aborted by the user');
-            } else {
+            signing = false;
+            if (response.success) {
               sendAddress(response.address, response.signature);
+            } else {
+              if (response.errorCode !== 'userAbort') {
+                alertUser(t('genericError'));
+                console.log('error: ' + response.errorMessage);
+              }
             }
           });
       }
-      if (message.type === V0MessageType.VerifyAddress) {
+      if (message.type === V0MessageType.VerifyAddress && !verifying) {
```

Comment:
Just noticed a small bug that allowed queuing several address verification. Fixed it before merging